plugins/node/opentelemetry-instrumentation-pg/src/semantic-conventions.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * These constants are considered experimental exports of `@opentelemetry/semantic-conventions`.
+ * They're being inlined until they're officially exported by `@opentelemetry/semantic-conventions`.
+ */
+
+/**
+ * The number of connections that are currently in state described by the `state` attribute
+ *
+ * @experimental This metric is experimental and is subject to breaking changes in minor releases of `@opentelemetry/semantic-conventions`.
```

Comment:
I copied these all in full, including the "experimental" warning. Happy to edit this to remove it desired.